tests/e2e/tests/web/regression/search-and-query/search-and-query-tab.e2e.ts

```diff
@@ -0,0 +1,176 @@
+let keyNames: string[];
+let indexName1: string;
+let indexName2: string;
+
+fixture `Autocomplete for entered commands in search and query`
+    .meta({ type: 'regression', rte: rte.standalone })
+    .page(commonUrl)
+    .beforeEach(async() => {
+        await databaseHelper.acceptLicenseTermsAndAddDatabaseApi(ossStandaloneConfig);
+        indexName1 = `idx1:${keyName}`;
+        indexName2 = `idx2:${keyName}`;
+        keyNames = [`${keyName}:1`, `${keyName}:2`, `${keyName}:3`];
+        const commands = [
+            `HSET ${keyNames[0]} "name" "Hall School" "description" " Spanning 10 states" "class" "independent" "type" "traditional" "address_city" "London" "address_street" "Manor Street" "students" 342 "location" "51.445417, -0.258352"`,
+            `HSET ${keyNames[1]} "name" "Garden School" "description" "Garden School is a new outdoor" "class" "state" "type" "forest; montessori;" "address_city" "London" "address_street" "Gordon Street" "students" 1452 "location" "51.402926, -0.321523"`,
+            `HSET ${keyNames[2]} "name" "Gillford School" "description" "Gillford School is a centre" "class" "private" "type" "democratic; waldorf" "address_city" "Goudhurst" "address_street" "Goudhurst" "students" 721 "location" "51.112685, 0.451076"`,
+            `FT.CREATE ${indexName1} ON HASH PREFIX 1 "${keyName}:" SCHEMA name TEXT NOSTEM description TEXT class TAG type TAG SEPARATOR ";" address_city AS city TAG address_street AS address TEXT NOSTEM students NUMERIC SORTABLE location GEO`,
+            `FT.CREATE ${indexName2} ON HASH PREFIX 1 "${keyName}:" SCHEMA name TEXT NOSTEM description TEXT class TAG type TAG SEPARATOR ";" address_city AS city TAG address_street AS address TEXT NOSTEM students NUMERIC SORTABLE location GEO`
+        ];
+
+        // Create 3 keys and index
+        await browserPage.Cli.sendCommandsInCli(commands);
+    })
+    .afterEach(async() => {
+        // Clear and delete database
+        await apiKeyRequests.deleteKeyByNameApi(keyName, ossStandaloneConfig.databaseName);
+        await browserPage.Cli.sendCommandsInCli([`DEL ${keyNames.join(' ')}`, `FT.DROPINDEX ${indexName1}`]);
+        await browserPage.Cli.sendCommandsInCli([`DEL ${keyNames.join(' ')}`, `FT.DROPINDEX ${indexName2}`]);
+        await databaseAPIRequests.deleteStandaloneDatabaseApi(ossStandaloneConfig);
+    });
+test('Verify that tutorials can be opened from Workbench', async t => {
+    const search = await browserPage.KeysInteractionPanel.setActiveTab(KeysInteractionTabs.SearchAndQuery);
+    await t.click(search.getTutorialLinkLocator('sq-exact-match'));
+    await t.expect(search.InsightsPanel.sidePanel.exists).ok('Insight panel is not opened');
+    const tab = await browserPage.InsightsPanel.setActiveTab(ExploreTabs.Tutorials);
+    await t.expect(tab.preselectArea.textContent).contains('EXACT MATCH', 'the tutorial page is incorrect');
+});
+test('Verify full commands suggestions with index and query for FT.AGGREGATE', async t => {
```

Comment:
can we split the test on part?
e.g. about tooltip
or on separate entered words